src/main/java/com/google/sps/data/Trip.java

```diff
@@ -10,17 +10,24 @@
 public abstract class Trip {
```

Comment:
I understand right now it's more efficient to directly associate Trips with their Hotels in order to reduce the number of database calls, but as Hotel info becomes more rich we should consider separating the two for the sake of more clearly defined entities as well as hierarchy i.e. Trip -> Hotel and Trip -> TripLocation

Reply:
This will also be more scalable in the future

Reply:
Also if we end up tackling multiple city trips that require multiple hotels, this will become necessary anyway.

Reply:
True that. I didn't separate Hotel into its own entity/value class yet since we only have one hotel for a trip. If we ever need a collection, with different dates and stuff, we'll go with the strategy you mentioned.